python/cudnn/linear_attention/frost/kernel/kda_bprop_f16.py

```
@@ -682,7 +645,7 @@ def super_mma_warp(
 
             lpow_a0, lpow_a1, lpow_a2, lpow_a3 = l_a0, l_a1, l_a2, l_a3
             mov_lpow0, mov_lpow1, mov_lpow2, mov_lpow3 = movmatrix_16b(l_a0), movmatrix_16b(l_a1), movmatrix_16b(l_a2), movmatrix_16b(l_a3)
-            for neumann_round in cutlass.range_constexpr(3):
+            for neumann_round in cutlass.range(3, unroll=1):
```

Ruff (0.16.2)

[warning] Loop control variable `neumann_round` not used within loop body

Rename unused `neumann_round` to `_neumann_round`

(B007)